presto-main/src/main/java/com/facebook/presto/server/StatementResource.java

```diff
@@ -210,6 +213,19 @@ private static Response getQueryResults(Query query, Optional<Long> token, UriIn
         query.getResetSessionProperties().stream()
                 .forEach(name -> response.header(PRESTO_CLEAR_SESSION, name));
 
+        // add added prepare statements
+        query.getAddedPreparedStatements().entrySet().stream()
+                .forEach(entry -> {
```

Comment:
This stream isn't doing anything other than iterating, so write it as a normal for-each loop